dali/driver/hasseb.py

```diff
@@ -13,181 +13,236 @@
 from dali.frame import ForwardFrame
 from dali.exceptions import ResponseError, MissingResponse
 
+from dali.gear.general import *
+
+from PyQt5.QtWidgets import QApplication
```

Comment:
IMHO we should not depend on Qt...

Reply:
Qt is required only when using graphical user interface. If graphical user interface is not used the two QT lines can be commented out. Any suggestions how this can be done without manually commenting the two lines out.

Reply:
Make it an anonymous callback, that is setup on initialisation

Reply:
I need to figure this out

Reply:
All QT dependencies removed from hasseb.py driver